pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -374,11 +374,11 @@ let
       sha256 =
         (
           if cudaSupport then
-            { x86_64-linux = "sha256-VGNMf5/DgXbgsu1w5J1Pmrukw+7UO31BNU+crKVsX5k="; }
+            { x86_64-linux = ""; }
           else
             {
-              x86_64-linux = "sha256-uOoAyMBLHPX6jzdN43b5wZV5eW0yI8sCDD7BSX2h4oQ=";
-              aarch64-linux = "sha256-+SnGKY9LIT1Qhu/x6Uh7sHRaAEjlc//qyKj1m4t16PA=";
+              x86_64-linux = "";
+              aarch64-linux = "";
```

Comment:
Is this normal?

Reply:
Yes, this is a WIP. I am stuck on https://github.com/NixOS/nixpkgs/pull/318995#issuecomment-2176902636.

I will mark this PR as draft.